scripts/compile/compile-stage.ps1

```diff
@@ -48,15 +48,6 @@ if (!$?) {
     Exit 1
 }
 
-# Build the projects that we are going to ship as nuget packages
-$ProjectsToPack | ForEach-Object {
```

Comment:
We cannot use the dotnet pack build because those won't be signed. The whole reason to have them separate is that we build them first, sign them and then run dotnet pack passing the folder where pack can find the binaries. Revert this.

Look at scripts/packages/packagesToPack.ps1 to see where this variable is defined. This is only a powershell script because we only generated the nupkgs in windows.